.github/workflows/manual_tests.yml

```diff
@@ -13,10 +13,10 @@ on:
         type: choice
         description: 'Python version'
         options:
-        - '3.8'
```

Comment:
I guess we don't use this workflow at all, so we might think about getting rid of it. But what is the reason for removing `3.8` now?